### PR TITLE
Latest SDKs plus code to support FileWatcherType for ApiHub file triggers.

### PR DESCRIPTION
**Pull request by emilcicos**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@emilcicos__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (emilcico). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by fabiocav** [comment]
Looks good to me. 👍

File: src/WebJobs.Script/Description/Binding/ApiHubBindingMetadata.cs [hunk 1]
```diff
@@ -15,6 +15,8 @@ public class ApiHubBindingMetadata : BindingMetadata
 
         public int PollIntervalInSeconds { get; set; }
 
+        public FileWatcherType FileWatcherType { get; set; }
```
Comment (mathewc) [inline src/WebJobs.Script/Description/Binding/ApiHubBindingMetadata.cs]: Shouldn't this have a `[JsonProperty(typeof(StringEnumConverter))]` on it?

Comment (emilcicos) [inline src/WebJobs.Script/Description/Binding/ApiHubBindingMetadata.cs]: Yes, will add [JsonConverter(typeof(StringEnumConverter))]

**Comment by mathewc** [comment]
Looks good - I'll merge this in this morning.

**Comment by davidebbo** [comment]
Is this something we want to deploy soon, or cook in dev branch for a while?

**Comment by galiniliev** [comment]
@davidebbo Since the portal is out in -next, would be great to have the full E2E story out. this is the only thing that is stopping us to send a note to the advisory board. It would be great if we can get it out today or tomorrow

**Comment by mathewc** [comment]
Merged.

**Pull request merged**